src/microbe_stage/MicrobeAI.cs

```diff
@@ -515,6 +519,10 @@ private void EngagePrey(Vector3 target, Random random, bool engulf)
         {
             SetMoveSpeed(Constants.AI_BASE_MOVEMENT);
         }
+
+        // Predators can use slime jets as an ambush mechanism
+        if (RollCheck(SpeciesAggression, Constants.MAX_SPECIES_AGGRESSION, random))
+            SecreteSlime();
```

Comment:
Given that you are out of the previous `if/else` statement, if a predator can shoot toxins, it may end up not moving towards the prey but only jetting towards them. I'm not sure such a behavior is desired?